trans/train.py

```diff
@@ -251,14 +301,13 @@ def main(args: argparse.Namespace):
 
 
 if __name__ == "__main__":
-
     logging.basicConfig(level="INFO", format="%(levelname)s: %(message)s")
 
     parser = argparse.ArgumentParser(
         description="Train a g2p neural transducer.")
 
     '''parser.add_argument("--dynet-seed", type=int, required=True,
-                        help="DyNET random seed.")''' # TODO: how to replace if at all?
+                        help="DyNET random seed.")'''  # TODO: how to replace if at all?
```

Comment:
We should probably eventually replace this with `torch` random seed.
https://pytorch.org/docs/stable/notes/randomness.html

Reply:
Absolutely, this is part of the changes of PR #2.